dspace-api/src/main/java/org/dspace/core/Context.java

```diff
@@ -595,4 +596,8 @@ protected void finalize() throws Throwable
     public void shutDownDatabase() throws SQLException {
         dbConnection.shutdown();
     }
+    
+    public void clearCache() throws SQLException {
+    	((Session)this.getDBConnection().getSession()).clear();
```

Comment:
I am just getting back to this.  getDBConnection() has package access, so I will keep the clearCache method in Context.